apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -76,6 +79,14 @@
 
     private static final String FILE_ATTACHMENT_KEY = "fileAttachment";
 
+    private static Long[] mLatencyValues = {
```

Comment:
I put a comment regarding the type of interval which is currently `int`. We should consider to refactor the return type to long. Just my opinion and not blocking anything.

Reply:
Assume we don't refactor the return type,
can we use int[] and put int values to the array when instantiate it?

Reply:
@jaelim-ms can you clarify this? Do you suggest creating an array with int but have long type?

Reply:
We will refactor the method that I mentioned [here](https://github.com/microsoft/AppCenter-SDK-Android/pull/1145#discussion_r282244350).
However a setter for transmission interval still takes an int value so the change would be determined by the setter. If the setter remains to take an int, this would be an int array.
https://github.com/microsoft/AppCenter-SDK-Android/pull/1145/files#diff-26bac3e4c1db2d55e1ecbb36aba942d7R922